core/che-core-api-core/src/main/java/org/eclipse/che/api/core/Page.java

```diff
@@ -91,6 +91,11 @@ public class Page<ITEM_T> {
   private final long totalCount;
   private final List<ITEM_T> items;
 
+  final String nonNullItemsString = "Required non-null items";
+  final String nonNegativeValueOfItems = "Required non-negative value of items before";
+  final String positiveValueOfPageSize = "Required positive value of page size";
+  final String nonNegativeValueOfTotalItems = "Required non-negative value of total items";
```

Comment:
Please rework to the [java constant style](https://en.wikibooks.org/wiki/Java_Programming/Coding_conventions#:~:text=Constants%20should%20be%20written%20with,%22%20and%20%22complex%22%20types.): 
```java
private static final String NON_NEGATIVE_VALUE_OF_TOTAL_ITEMS = "Required non-negative value of total items"
```

Reply:
That is a Good idea. ^^
Thanks